samples/exemplarchat/README.md

```diff
@@ -2,3 +2,21 @@
 
 This is a sample app that makes use of Misk. It demonstrates both web sockets and Misk's recommended
 patterns for embedded web UIs.
+
+## Usage
+
+1. Build the React app in the `web` directory.
+    ```
+    cd samples/exemplarchat/web
+    yarn
+    yarn run build
+    ```
+2. Build the project using gradle.
+    ```bash
+    ./gradlew :samples:exemplarchat:build    
+    ```
+2. Run the sample.
+    ```bash
+    java -jar samples/exemplarchat/build/libs/exemplarchat-0.8.0-SNAPSHOT.jar    
```

Comment:
I haven't figured out a way to run directly from Gradle with the shaded resources from `web`.  Perhaps adding the web's build directory to the `sourceSets.main.resources`?  But then how do I get Gradle to build the React app as part of that?

Reply:
We could use a gradle plugin to run `npm install`

Reply:
+1, to adding the built web to sourceSets.main.resources